spring-boot-project/spring-boot-docs/src/docs/antora/modules/how-to/pages/deployment/cloud.adoc

```diff
@@ -244,7 +244,7 @@ For more details, see https://devcenter.heroku.com/articles/deploying-spring-boo
 https://www.openshift.com/[OpenShift] has many resources describing how to deploy Spring Boot applications, including:
 
 * https://blog.openshift.com/using-openshift-enterprise-grade-spring-boot-deployments/[Using the S2I builder]
-* https://access.redhat.com/documentation/en-us/reference_architectures/2017/html-single/spring_boot_microservices_on_red_hat_openshift_container_platform_3/[Architecture guide]
+* https://github.com/RHsyseng/spring-boot-msa-ocp[Architecture guide]
```

Comment:
Thanks for the PR, but I don't think this is a good replacement. Instead, let's remove the item from the list please.